templates/commands.conf.j2

```diff
@@ -191,7 +191,7 @@ object CheckCommand "raid" {
   command = [PluginDir + "/check_raid.pl" ]
 }
 
-object CheckCommand "ceph" {
+object CheckCommand "ceph-custom" {
```

Comment:
If you rename the command you'll most likely need to change the apply rule as well (https://github.com/SOSETH/monitoring-server/blob/master/templates/icinga2/services.conf.j2). However, since this is the check for CEPH below Luminous (aka no-manager-CEPH), I'll have a look if someone still uses this, otherwise, we might as well drop the whole check.

Reply:
As far as I can see, everybody has now upgraded to at least CEPH Luminous. Can you remove this command from the MR?